src/main/java/net/snowflake/ingest/streaming/internal/BlobLatencies.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+package net.snowflake.ingest.streaming.internal;
+
+import com.codahale.metrics.Timer;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.concurrent.TimeUnit;
+
+/** Latency information for a blob */
+class BlobLatencies {
+  public static final long DEFAULT_BLOB_LATENCY = -1;
+
+  private long buildLatencyMs;
+  private long uploadLatencyMs;
```

Comment:
Should these be renamed to use `duration` instead of `latency`?

Reply:
That would make more sense, renamed to duration